### PR TITLE
[RW-6058]: Batch load/upload user report

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:
This is similar to what we did for workspace table. Those two are the most problematic tables. 

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java [hunk 1]
```diff
@@ -60,6 +62,9 @@ public void collectRecordsAndUpload() {
     reportingQueryService
         .getWorkspacesStream()
         .forEach(b -> reportingUploadService.uploadBatchWorkspace(b, captureTimestamp));
+    reportingQueryService
+        .getUserStream()
+        .forEach(b -> reportingUploadService.uploadBatchWorkspace(b, captureTimestamp));
```
Comment (calbach) [inline api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java:67]: hm.. does this compile? uploadBatch**Workspace**

Comment (yonghaoy) [reply api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java:67]: nope

Comment (calbach) [reply api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java:67]: I mean - presumably this is a typo and should have been `uploadBatchUser` - why does CI pass though with this bug?

Comment (yonghaoy) [reply api/src/main/java/org/pmiops/workbench/reporting/ReportingServiceImpl.java:67]: It is compiled by CI, but that is a mistake. Sorry for the confusion, I let getUserStream returns Workspaces, and just realize we don't have test for `getUserStream()`
Added one for that.

**Review by calbach — APPROVED** [review]
Thanks

**Pull request merged**